www/js/localstorage.js

```diff
@@ -0,0 +1,36 @@
+'user strick';
+
+angular.module('emission.main.localstorage',['angularLocalStorage'])
```

Comment:
If we follow my pattern, I would have moved this functionality into `CalorieCalc` (see consent code for example). We would not have such a module (which is why we haven't had one so far). 

The main reason for reading and writing in the place where it is used, as opposed to a separate module is the encapsulation of the data model. If we wanted to change the name of one of the fields for the calorie calculation, for e.g. - `heightUnit` to `height_unit`, we would need to change it in both `CalorieCalc` and here. But if we move this function into the `CalorieCalc`, we only need to change it in one place.

So if somebody new wants to touch this code 5 years later, they won't have to know that they need to modify two places.